changes/v4.4.12-en.md

```diff
@@ -6,6 +6,10 @@
 
 - Users can define the `externalTrafficPolicy` of service in EMQX Helm Chart [#9527](https://github.com/emqx/emqx/pull/9527).
 
+- Start releasing packages for Amazon Linux 2 [#9498](https://github.com/emqx/emqx/pull/9498).
+
+- Start building MacOS packages for Apple Silicon hadrdware [#9558](https://github.com/emqx/emqx/pull/9558).
```

Comment:
tr:
```
新增 Apple Silicon M1/M2 安装包发布 [#9558](https://github.com/emqx/emqx/pull/9558)。
```

Reply:
@zmstone and for amazon linux? (forgot to add it last time)